content/md/first-steps/manage-your-attributes.md

```diff
@@ -268,31 +268,97 @@ Unlike products you cannot restore or revert an attribute’s version.
 
 # Delete an attribute
 
+There are two ways to remove an attribute:
+* Through the attribute edit form
+* Through the delete shortcut in the grid
+
+You can also choose if you want to loose or to keep the data used by this attribute.
```

Comment:
This is ambiguous. The choice is not proposed in the PIM. I would write something like 
> When removing an attribute, all values related to this attribute are lost. If you want to keep this attribute values, please read the part "Delete an attribute and keeping the related data".

Reply:
Good point, thanks.